deegree-datastores/deegree-featurestores/deegree-featurestore-sql/src/main/java/org/deegree/feature/persistence/sql/SQLFeatureStoreTransaction.java

```diff
@@ -556,7 +560,7 @@ public List<String> performInsert( FeatureCollection fc, IDGenMode mode )
         for ( FeatureInspector inspector : inspectors ) {
             for ( Feature f : features ) {
                 // TODO cope with inspectors that return a different instance
-                inspector.inspect( f );
+                inspector.inspect( f, this );
```

Comment:
Comment https://github.com/deegree/deegree3/pull/1290#issuecomment-1083354343
> Postponed because the actual change to use the new interface was pushed to another pr, but was supposed to be included in this one.

After a new review, it turned out that all affected changes are already included.